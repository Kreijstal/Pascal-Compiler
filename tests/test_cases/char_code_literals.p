program char_code_literals;

var
  ch: char;

begin
  ch := #9;
  writeln('Tab character assigned');

  ch := #10;
  writeln('Line feed character assigned');

  ch := #13;
  writeln('Carriage return character assigned');

  writeln('All character codes assigned successfully');
end.
