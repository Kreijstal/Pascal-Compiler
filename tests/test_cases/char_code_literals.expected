Tab character assigned
Line feed character assigned
Carriage return character assigned
All character codes assigned successfully
